### DISCUSSION TITLE
Feature request: allow private repos to be conditionally pinned?

### DISCUSSION THREAD

**Discussion by andrewschultz** [community/community#22263] [New to GitHub]
<p>I recognize this might be pretty low on the priority list, but I’ve started tinkering with private repos (which are great) and I realized that the front page had two different sorts of uses.</p>
<ol>
<li>a quick reference for when I want to go to my most important project</li>
<li>a way to show people what I’ve done so far</li>
</ol>
<p>The thing is that 1 and 2 are sort of at odds. The repos I want the easiest access to are the private ones I haven’t finished yet, while the repos I want to show to other people are the ones that are done and have a few stars and such.</p>
<p>Now I have an easy workaround, which is to type alt-d and one word from my private repo name into the address bar in my web browser. The first suggestion is usually the one that works. And since I only need three keystrokes, this is hardly an inconvenience.</p>
<p>But it would be neat to have a conditional ordering, e.g.</p>
<p>Private 1, private 2, public 1, public 2, private 3, public 3, public 4, public 5, public 6.</p>
<p>Then I would see</p>
<p>pri1 pri2</p>
<p>pub1 pub2</p>
<p>pri3 pub3</p>
<p>And I would be reminded of what I most want and need to work on.</p>
<p>Visitors, though, would see</p>
<p>pub1 pub2</p>
<p>pub3 pub4</p>
<p>pub5 pub6</p>
<p>(The repos with the most work done)</p>
<p>And a friend with access to, say, private repo 2 would see</p>
<p>pri2 pub1</p>
<p>pub2 pub3</p>
<p>pub4 pub5</p>
<p>Is this practical or doable? Or is there a better work-around than just typing in part of the repo name in the browser address bar?</p>
<p>It seems like something that would be neat for the end user, but it’s just a convenience as opposed to a core change. Still, I wanted to suggest it in case it had more uses than I thought.</p>

**Comment by AndreaGriffiths11** [comment, marked as answer]
<p>Hey <a class="mention" href="/andrewschultz">@andrewschultz</a>,</p>
<p>Thanks for reaching out! There is an internal issue discussing this very feature and I’ll add your feedback to it. Thanks again for taking the time to post. In the meantime, we’re looking forward to seeing you around!</p>

**Comment by andrewschultz** [comment]
<p>Thanks! It’s good to know that my requests are in tune with what was planned in the big picture. I always wory they might be too fiddly.</p>
<p>In the meantime, I’ll tack on a script for Windows Task Scheduler to launch my two private repos in addition to my main github page in th evening,  to help make sure I do a bit of work each day.</p>
